addons/binding/org.openhab.binding.upb/src/main/java/org/openhab/binding/upb/handler/UPBBaseHandler.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.upb.handler;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.upb.UPBBindingConstants;
+import org.openhab.binding.upb.internal.MessageBuilder;
+import org.openhab.binding.upb.internal.UPBMessage;
+import org.openhab.binding.upb.internal.converter.StateConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Base implementation for handling commands sent to UPB things.
+ *
+ * @author Chris Van Orman
+ * @since 2.0.0
+ *
+ */
+public abstract class UPBBaseHandler extends BaseThingHandler implements UPBMessageListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(UPBBaseHandler.class);
```

Comment:
Logger should NOT be static according to the guidelines. [[source](http://docs.openhab.org/developers/development/guidelines.html#e-logging)]